app/views/people/_new_search_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_for @search, as: :people_search, :url => search_people_path, :html => {:class => 'search', :method => :get} do |f| %>
+  <fieldset>
+    <legend>Search people</legend>
```

Comment:
Could you make this translatable and remove the colons from the field labels? Also, if you could convert it from hash rockets, that would be great